File: DSA-IN-C++/BasicMath/countDigit.cpp
#include <iostream>
using namespace std;

int main(){
    int n,count=0;
    cout<<"Enter the number:";
    cin>>n;
    while(n>0){
        n=n/10;
        count++;
    }
    cout<<"Total number of digits: "<<count<<endl;
    return 0;
}

File: DSA-IN-JAVA-Practice/Basic_Math/Count_Digit.java
import java.util.*;
class Count_Digit{
public static void main(String [] args){
    System.out.println("Enter a number: ");
    Scanner sc=new Scanner(System.in);

      int n=sc.nextInt();
      int count=0;
      while(n>0){
       n=n/10;
       count++;

      }
      System.out.println(count);
}
}